foundation/foundation-security/pom.xml

```diff
@@ -15,7 +15,6 @@
 	<description>Contains the modules used for security</description>
 
 	<modules>
-		<module>foundation-policy-decision-point</module>
```

Comment:
I: Other than docs/javadocs, the classes/files in this module were identical to `extensions-policy-decision-point-service`. I believe these ended up as duplicates during the move from isolated repos to the monorepo. (Two separate repos had the same classes and got pulled into different places in the monorepo.)